ovirt-engine-dwh.spec.in

```diff
@@ -316,6 +316,11 @@ fi
 %{_sysconfdir}/grafana/conf/
 
 %changelog
+* Fri Mar 18 2022 Martin Perina <mperina@redhat.com> - 4.5.1
+- Use fasterxml for ResumeUtil json serialization
```

Comment:
Add bug number

Reply:
Hmm, bug number was only in a PR, but not in commit, we really need to have some more guidelines how to work with GH :-(

Updating

Reply:
+1 I will add it from now on in both commit and pr